src/test/FunctionTest/InsertCn.cs

```diff
@@ -84,15 +95,15 @@ public void TestSTable()
             IntPtr _res = IntPtr.Zero;
             string tableName = "cn_insert_nchar_stable";
             var colData = new List<Object>{1637064040000,1,"涛思数据",
-            1637064041000,2,"涛思数据taosdata",
-            1637064042000,3,"TDegnine涛思数据",
-            1637064043000,4,"4涛思数据",
-            1637064044000,5,"涛思数据5",
-            1637064045000,6,"taos涛思数据6",
-            1637064046000,7,"7涛思数据taos",
-            1637064047000,8,"8&涛思数据taos",
-            1637064048000,9,"&涛思数据taos9"
-            };
+                    1637064041000,2,"涛思数据taosdata",
+                    1637064042000,3,"TDegnine涛思数据",
+                    1637064043000,4,"4涛思数据",
+                    1637064044000,5,"涛思数据5",
+                    1637064045000,6,"taos涛思数据6",
+                    1637064046000,7,"7涛思数据taos",
+                    1637064047000,8,"8&涛思数据taos",
+                    1637064048000,9,"&涛思数据taos9"
+                    };
```

Comment:
indent error?

Reply:
This is formated by dotnet CLI. Maybe  this is conflict with dotnet convention.